code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -159,6 +162,24 @@
 				affect_touch(M, alien, removed)
 	if(overdose && (volume > overdose * M?.species.chemOD_threshold) && (active_metab.metabolism_class != CHEM_TOUCH && !can_overdose_touch))
 		overdose(M, alien, removed)
+	if(M.species.allergens & allergen_type)	//uhoh, we can't handle this!		

```

Comment:
I feel like multiplying each value just once is absolutely the move here, mostly for cleanliness.
```suggestion
	if(M.species.allergens & allergen_type)	//uhoh, we can't handle this!		
		var/damage_severity = M.species.allergen_damage_severity*allergen_factor
		var/disable_severity = M.species.allergen_disable_severity*allergen_factor
```

Reply:
Good idea.